blockindex/bloomfilterindexer.go

```diff
@@ -32,9 +34,18 @@ const (
 	CurrentHeightKey = "CurrentHeight"
 )
 
+const (
+	_maxBlockRange = 1e6
+	_workerNumbers = 5
+)
+
 var (
 	// TotalBloomFilterNamespace indicates the kvstore namespace to store total ranges
 	TotalBloomFilterNamespace = []byte("TotalBloomFilters")
+
+	errRangeTooLarge = errors.New("block range is too large")
+
+	_queryTimeout = 10 * time.Second
```

Comment:
feel this might be too small? or do we want to be very strict limiting it to under 10s.
it would be helpful if we have real data from API queries to determine a proper timeout value